src/main/java/org/killbill/billing/plugin/adyen/core/resources/AdyenCheckoutServlet.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020-2023 Equinix, Inc
+ * Copyright 2014-2023 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.plugin.adyen.core.resources;
+
+import com.google.inject.Inject;
```

Comment:
We could use `javax.inject.Inject;` instead of google's `Inject`

Reply:
okay, but just to confirm again, other plugins like [Stripe ](https://github.com/killbill/killbill-stripe-plugin/blob/2371cf9e6fa39c660dd26543319e2a5d73fd935b/src/main/java/org/killbill/billing/plugin/stripe/StripeCheckoutServlet.java#L45) use google `Inject`. So should we still use `javax.inject.Inject` or use `com.google.inject.Inject` for consistency sake?

Reply:
Yes, for consistency. 

It is not a must requirement on something, but since work on https://github.com/killbill/killbill/issues/1615 , I gradually also move `com.google.inject.*` to `javax.inject.*` whenever possible. I think I had discussion in support with Pierre, and AFAIR, he agrees on this. 

Stripe part you mention is also good example that we should change this gradually. And since those library are compatible, we can use one in one project/module and the other in another project/modules with minimum to no problem (so far, I found no problem with this in killbill).